common/test/db_cache/bok_choy_schema.sql

```diff
@@ -623,6 +623,20 @@ CREATE TABLE `certificates_certificategenerationcoursesetting` (
   KEY `certificates_certificategenerationcoursesetting_b4b47e7a` (`course_key`)
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8;
 /*!40101 SET character_set_client = @saved_cs_client */;
+DROP TABLE IF EXISTS `certificates_certificatehtmlviewconfiguration`;
```

Comment:
hmm.. regeneration of a file via script should be automated.  Is TestEng aware of this issue?
What impact is there if the file is not regenerated?  Will bokchoy tests fail?